configs/components/echam/echam.yaml

```diff
@@ -352,6 +352,9 @@ choose_scenario:
                         swflux: 1950swflux
                         volcir: 1950volcir
                         volcsw: 1950volcsw
+        # new scenario NUDG for ECHAM6-wiso nudged simulations (M. Werner, 09/2022)
+        NUDG:
+                dataset: r0007
```

Comment:
Doesn't `r0008` have nudging files? If it does, but you want to use `r0007` for a given experiment, wouldn't this belong to the runscript instead, and not to a `NUDG` scenario?

Reply:
Which files are directly needed for nudging? Maybe it is an idea to add these as a separate listing instead of from a dataset? That would gain some independent control of the dataset and the nudging files, unless there is a good reason to link these two together.